sidebars.js

```diff
@@ -56,6 +56,7 @@ module.exports = {
       label: 'Advanced',
       collapsible: false,
       items: [
+        'rest-api',
```

Comment:
I wonder if it might make sense to add a link to this page (not a docs reference but an absolute link) to the "extend" sidebar as well. It's not quite an extension, but some developers might look there anyway.

Reply:
Yeah I wasn't sure about this.

It would make sense for a "REST API" page to exist on both regular documentation and extension documentation, where one explains how to use the API and the other explains how to extend the API.

But at the same time extensions will also use the existing REST API so it does make sense to cross-link.

Maybe the page should be titled more clearly, something like "Consuming REST API consumption" so when there's an eventual "Extending REST API" it's more clear which one is meant for who?

Reply:
I agree that clearing it up with "Consuming" and "Extending" would be very helpful in the long run.

Reply:
Should I change just the title or also the URL?

I suppose the URL structure will change later on if we introduce a proper endpoint documentation, and we might move/refactor this document into the new pages.

Reply:
Ok so there is already a "API and Data Flow" page that handles this subject in the Extend section, so I don't think it makes sense to add more pages or links in the sidebar. Maybe rename the name or URL of `extend/api.md` ?

For now I have added links between the 2 pages via info blocks.